packages/webpack/src/resource_loader.ts

```diff
@@ -0,0 +1,116 @@
+import {ResourceLoader} from '@angular/compiler';
+import {readFileSync} from 'fs';
+import * as vm from 'vm';
+import * as path from 'path';
+
+const NodeTemplatePlugin = require('webpack/lib/node/NodeTemplatePlugin');
+const NodeTargetPlugin = require('webpack/lib/node/NodeTargetPlugin');
+const LoaderTargetPlugin = require('webpack/lib/LoaderTargetPlugin');
+const SingleEntryPlugin = require('webpack/lib/SingleEntryPlugin');
+
+
+
+export class WebpackResourceLoader implements ResourceLoader {
```

Comment:
@TheLarkInn  I don't know what you're talking about, this was a requirement from the design phase. This resource loader compiles the resources asked for the component and as such will use the loaders.